### DISCUSSION TITLE
MQTT – angezeigte Werte für temporäres und permanentes Limit immer gleich

### DISCUSSION THREAD

**Discussion by Github-Nutzer** [tbnobody/OpenDTU#1246] [General]
Hallo!

Meine OpenDTU ist via MQTT in HomeAssistant eingebunden. Nun ist mir beim Experimentieren im Hinblick auf Nulleinspeisung aufgefallen, dass bei temporären Limitänderungen der Wechselrichterleistung (zumindest lt. den angezeigten MQTT-Werten in HA) immer auch das dauerhafte Limit mitgeändert wird. Das irritiert mich jetzt ein wenig – Stichwort EEprom-Verschleiß.

Verständnisproblem, Anzeigefehler, …?

Danke!

**Comment by turrican944** [comment]
Moin
Das ist nur die Anzeige der WR gibt (wahrscheinlich) das Limit nur einmal aus. Wenn du per Homeassistant ein temporäres Limit setzt wird das nicht gespeichert. Kannst du ausprobieren, Limit setzen und dann auf Inverter Reset gehen, danach ist das dauerhafte wieder da.
Die Opendtu kann wenn der WR nur ein Limit zurück meldet ja auch nicht unterscheiden ob das nun das dauerhafte oder das temporäre ist. Gut die Software könnte sich merken welches als letztes gesetzt wurde, aber das mach dann ggf. nach einem Inverter Neustart noch mehr Verwirrung.
